### PR TITLE
Large set of updates for multilepton ICHEP analyses

### PR DESCRIPTION
**Pull request by peruzzim**
Please see here for diff and comments, split for directories:

JEC files: gpetruc/cmgtools-lite#1
samples, utilities, HZZ: gpetruc/cmgtools-lite#2
analysis and plotting framework: gpetruc/cmgtools-lite#3
utility files (scale factors, fake rates etc.): gpetruc/cmgtools-lite#4
SUSY multilepton: gpetruc/cmgtools-lite#5
SUSY SOS: gpetruc/cmgtools-lite#6
TTH multilepton: gpetruc/cmgtools-lite#7

### CODE REVIEW

**Comment by gpetruc** [comment]
Since this PR touches a lot of things, can people please test that it does not break your setup?
@mschoene @mariadalfonso @emanueledimarco @clelange @steggema @mdunser @aelwood

**Comment by peruzzim** [comment]
Are there any reports on the compatibility of this PR with other setups?
We should aim at merging this on a short timescale. Thanks!

**Comment by steggema** [comment]
Looks good to me.

File: RootTools/python/samples/samples_13TeV_RunIISpring16MiniAODv2.py [hunk 1]
```diff
@@ -529,11 +535,14 @@
...
 
 ZZTo2L2Nu = kreator.makeMCComponent("ZZTo2L2Nu", "/ZZTo2L2Nu_13TeV_powheg_pythia8/RunIISpring16MiniAODv2-PUSpring16_80X_mcRun2_asymptotic_2016_miniAODv2_v0-v1/MINIAODSIM", "CMS", ".*root", 0.564)
 ZZTo2L2Q = kreator.makeMCComponent("ZZTo2L2Q", "/ZZTo2L2Q_13TeV_amcatnloFXFX_madspin_pythia8/RunIISpring16MiniAODv2-PUSpring16_80X_mcRun2_asymptotic_2016_miniAODv2_v0-v1/MINIAODSIM", "CMS", ".*root",  3.28)
 ZZTo2Q2Nu = kreator.makeMCComponent("ZZTo2Q2Nu", "/ZZTo2Q2Nu_13TeV_amcatnloFXFX_madspin_pythia8/RunIISpring16MiniAODv2-PUSpring16_80X_mcRun2_asymptotic_2016_miniAODv2_v0-v1/MINIAODSIM", "CMS", ".*root",  4.04)
+ZZTo2L2Nu = kreator.makeMCComponent("ZZTo2L2Nu", "/ZZTo2L2Nu_13TeV_powheg_pythia8/RunIISpring16MiniAODv2-PUSpring16_80X_mcRun2_asymptotic_2016_miniAODv2_v0-v1/MINIAODSIM", "CMS", ".*root", 0.564)
+ZZTo2L2NuM4to40_notau= kreator.makeMCComponentFromEOS('ZZTo2L2NuM4to40_notau', 'ZZllvv_aMCatNLO_mll4_40gev_miniAODSIMv2/', '/store/user/castello/susy/%s',".*notau.*root", 0.219)
```
Comment (clelange) [inline RootTools/python/samples/samples_13TeV_RunIISpring16MiniAODv2.py]: This points to files that don't exist, and crashes heppy for me - probably should be removed including following line and lines 582-3.
`RuntimeError: ERROR making component ZZTo2L2NuM4to40_notau: no files found under /store/user/castello/susy/ZZllvv_aMCatNLO_mll4_40gev_miniAODSIMv2/ matching '.*notau.*root'`

Comment (clelange) [reply RootTools/python/samples/samples_13TeV_RunIISpring16MiniAODv2.py]: I tested on 100 events, no changes found for VVResonances, so I'm fine with merging once this sample issue is fixed.

**Pull request merged**